chess/__init__.py

```diff
@@ -26,12 +26,17 @@
 import re
 
 
-COLORS = [ WHITE, BLACK ] = range(2)
+COLORS = [ WHITE, BLACK ] = list(range(2))
```

Comment:
In general adding list(...) around ranges or views is not required. We're just iterating over them anyway. This should also apply to all other cases.